File: Kurs C++/Zad6/stala.cpp
//
// Created by Mateusz Pater on 08.04.2016.
//

#include "stala.h"



double stala::oblicz(){
    return value;
}

std::string stala::opis() {

}

stala::stala(int value) : value(value) { }


int stala::getValue() {
    return value;
}

std::string stala::formatuj_wyrazenie() {
    return std::to_string(value);
}

File: Kurs Java/Zadanie 3/src/com/company/Stala.java
package com.company;

/**
 * author @pater
 */
public class Stala extends Wyrazenie {
    double value;

    public Stala(double value) {
        this.value = value;
    }

    @Override
    public double oblicz() {
        return value;
    }

    @Override
    public String toString() {
        return ""+value;
    }
}

